tests/test_auth_basic_auth.py

```diff
@@ -79,3 +82,16 @@ def test_valid_credential_twofactor_valid_otp(self):
 
         res = self.app.get(self.reachable_url, auth=self.user1.auth, headers={'X-OSF-OTP': _valid_code(self.TOTP_SECRET)})
         assert_equal(res.status_code, 200)
+
+    def test_valid_cookie(self):
+        cookie = self.user1.get_or_create_cookie()
+        self.app.set_cookie(settings.COOKIE_NAME, str(cookie))
+        res = self.app.get(self.reachable_url)
+        assert_equal(res.status_code, 200)
+
+    def test_expired_cookie(self):
+        self.session = SessionFactory(user=self.user1, date_created=(datetime.utcnow() - timedelta(seconds=settings.OSF_SESSION_TIMEOUT)))
+        cookie = self.user1.get_or_create_cookie()
+        self.app.set_cookie(settings.COOKIE_NAME, str(cookie))
+        res = self.app.get(self.reachable_url)
+        assert_equal(res.status_code, 302)
```

Comment:
It might be good to `assert_in('login', res.location)` as well, but it's probably fine as is.